index.html

```diff
@@ -1,23 +1,171 @@
-  <!-- link your style.css using <link> tag -->
-  <title>Learning how to learn</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Learning how to learn</title>
+    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="pages/index.css">
 </head>
 <body>
   <div class="page">
     <header class="header">
+      <div class="logo_place_header logo"></div>
+        <h1 class="header__title">Learning How to Learn</h1>
+        <div class="header__square-pic rotation"></div>
+          <p class="header__subtitle">
+                The most effective, cutting-edge learning  techniques that you can apply to your daily life
+          <a class="header__link" href="#anchor">Learn more &rarr;</a>
+          </p>
+        <img class="header__main-illustration" src="images/header-image.png" alt="">
     </header>
-    <main class="content">
-      <section class="description">
+    <main id="anchor" class="content">
+      <section class="two-columns">
+          <h2 class="two-columns__brief">
+            The main problem  with education
+          </h2>
+          <div class="two-columns__main-text">
+              <p class="two-columns__paragraph">
+                  Whether it's at school or university,
+                  the one thing no own ever teaches you is how to teach yourself.
+                  We often just end up drilling specific exercises and mindlessly
+                  memorizing information sso that when exam time comes around, we pass
+                  the test. But once the test is over, that information has outlived
+                  its usefulness,s and there's no guarantee that information will be
+                  retained. In the end, we're really just learning how to pass our tests.
+              </p>
+              <p class="two-columns__paragraph">
+                  <span class="two-columns__span-accent">
+                  Here's the real question:</span> does this approach of repetition and
+                  memorization help us truly understand the subject material?
+                  The academic consensus nowadays is that it doesn't. In fact, what's
+                  more important than any individual topic is knowing how to study, but
+                  for some reason, this point is often overlooked.
+                  So, what can we do to remedy this?
+              </p>
+              <p class="two-columns__paragraph">
+                  Well, there are various techniques and exercises that can be applied
+                  to the educational process to help make it more efficient and engaging.
+                  Keep scrolling to uncover some of the secrets of effective learning.
+              </p>
+          </div>
+      </section>
+      <section class="techniques">
+        <h2 class="section-title title_color_black">Learning Techniques</h2>
```

Comment:
I couldn't find any styles for `title_color_black.` If you're not using this selector, it's best to remove it in order to keep things clean.